flixel/system/debug/console/Console.hx

```diff
@@ -117,8 +117,10 @@ class Console extends Window
 		// openfl/openfl#1856
 		FlxG.stage.addEventListener(KeyboardEvent.KEY_DOWN, function(e:KeyboardEvent)
 		{
-			if (FlxG.debugger.visible && FlxG.game.debugger.console.visible && e.keyCode == Keyboard.TAB)
+			if (FlxG.debugger.visible && FlxG.game.debugger.console.visible && e.keyCode == Keyboard.TAB) {
```

Comment:
don't cuddle brackets
```suggestion
			if (FlxG.debugger.visible && FlxG.game.debugger.console.visible && e.keyCode == Keyboard.TAB)
			{
```

Reply:
also why add the bracket here in the first place, at first I assumed it was to include `e.preventDefault();`